src/components/NewMessageForm/AdvancedInput/AdvancedInput.vue

```diff
@@ -249,6 +250,15 @@ export default {
 		EventBus.$off('focusChatInput', this.focusInput)
 	},
 	methods: {
+		onBlur() {
+			// requires a short delay to avoid blocking click event handlers
+			// from vue-at which also have some delay in place...
+			// a setTimeout was recommended by the library author here:
+			// https://github.com/fritx/vue-at/issues/114#issuecomment-565777450
+			setTimeout(() => {
```

Comment:
`this.$next()` could help/work?

Reply:
no, nextTick is the equivalent of setTimeout with a value of zero
I even tried with 50 and it was not high enough...

I saw in the source code of the lib that it has some timeouts already in place.

Reply:
and even the lib author suggested to do that, which surprised me at first... probably he knew about the timeouts issue...